utilities/measurement_utility.cc

```diff
@@ -114,16 +146,52 @@ int hash_utility(string &input, string &policy_key_file, string &output) {
     printf("\n");
   }
 
-  free(to_hash);
+  free(to_hash_start);
   return 0;
 }
 
+/*
+ * parse_other_files_size() - Parse a list of comma-separated file names which
+ * should be included in the measurement. Get each file's size, and return
+ * an error if file could not be found.
+ *
+ * Parameters:
+ *  other_files      - (In) Comma-separated list of file names
+ *  other_files_list - (Out) Array of file names found
+ *  other_files_size - (Out) Array of each file's size
+ *
+ * Returns: total size of all files listed in 'other_files'
+ */
+int parse_other_files_size(string &        other_files,
+                           vector<string> &other_files_list,
+                           vector<int> &   other_files_size) {
+  stringstream ss(other_files);
+
+  int other_files_total_size = 0;
+  while (ss.good()) {
+    string substr;
+    getline(ss, substr, ',');
+    int this_files_size = file_size(substr);
```

Comment:
`file_size()` does not return an error if file is not found. It returns -1, so check for that, and print an error here.

Reply:
yes